src/main/webapp/search-interview.js

```diff
@@ -14,23 +14,35 @@
 
 function onSearchInterviewLoad() {
   const loginInfo = getLoginInfo();
-  loginInfo.then(supplyLogoutLinkOrRedirectHome); 
+  loginInfo.then(ifLoggedOutRedirectHome); 
   loginInfo.then(getUserOrRedirectRegistration);
 }
 
 // Queries Datastore for available interview times and renders them on the
 // page.
 function loadInterviews() {
-  const searchResultsDiv = document.getElementById("search-results");
-  searchResultsDiv.removeAttribute("hidden");
-  fetch(`/load-interviews?timeZoneOffset=${browserTimezoneOffset()}&position=${selectedPosition()}`)
-  .then(response => response.text())
-  .then(interviewTimes => {
-    interviewTimesDiv().innerHTML = interviewTimes;
-  });
+  const searchResultsDiv = document.getElementById('search-results');
+  searchResultsDiv.removeAttribute('hidden');
+  const role = selectedRole();
+  let servlet = '';
```

Comment:
Just FYI, this is one of those design decisions that would hurt a lot long term.

Basically, you're baking in an assumption that these two servlets accept the same request json and URL parameters. Without EndToEnd tests, this type of thing would break and only be seen in prod.

Reply:
I agree it would have to be decoupled as soon as they diverged. Claire asked for this here because there was so much repetition in the way we currently have it set up.